src/Content/Embed/index.tsx

```diff
@@ -148,13 +148,21 @@ function Embed({ embed, images }: EmbedProps) {
         />
       )}
       {images && images.length > 0 && (
-        <Styles.Images nImages={images.length as 1 | 2 | 3 | 4}>
-          {images.map((image) => (
-            <Styles.ImageGridImageContainer key={image.url}>
-              <EmbeddedImage embedImage={image} withMargin />
-            </Styles.ImageGridImageContainer>
-          ))}
-        </Styles.Images>
+        // <Styles.Images nImages={images.length}>
+        //   {images.map((image) => (
+        //     <Styles.ImageGridImageContainer key={image.url}>
+        //       <EmbeddedImage embedImage={image} withMargin />
+        //     </Styles.ImageGridImageContainer>
+        //   ))}
+        // </Styles.Images>
```

Comment:
Why not just remove it?

Reply:
I'd personally opt for making a const like `MAX_IMAGES`, then we can use images.slice(0, MAX_IMAGES)